app/utils/temporal.ts

```diff
@@ -0,0 +1,198 @@
+import { isDefined } from '@togglecorp/fujs';
+
+export function getDateSafe(value: Date | number | string) {
```

Comment:
Let's write tests for these functions.
Let's try tests with different timezones.